x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -380,10 +421,32 @@ export const LogRateAnalysisResults: FC<LogRateAnalysisResultsProps> = ({
           <FieldFilterPopover
             disabled={!groupResults || isRunning}
             disabledApplyButton={isRunning}
-            uniqueFieldNames={uniqueFieldNames}
+            disabledApplyTooltipContent={disabledFieldFilterApplyButtonTooltipContent}
+            helpText={fieldFilterHelpText}
+            itemSearchAriaLabel={fieldsButton}
+            popoverButtonTitle={fieldsButton}
+            uniqueItemNames={uniqueFieldNames}
             onChange={onFieldsFilterChange}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <ColumnFilterPopover
+            disabled={isRunning}
+            disabledApplyButton={isRunning}
+            disabledApplyTooltipContent={disabledColumnFilterApplyButtonTooltipContent}
+            helpText={columnsFilterHelpText}
+            itemSearchAriaLabel={columnSearchAriaLabel}
+            initialSkippedItems={skippedColumns}
+            popoverButtonTitle={columnsButton}
+            selectedItemLimit={1}
```

Comment:
I'm seeing a weird issue where if I open the popover, hit 'Deselect all', I then have to select 2 items (or sometimes more) to get the Apply button enabled:

<img width="562" alt="Screenshot 2024-06-06 at 10 40 43" src="https://github.com/elastic/kibana/assets/7405507/5d1a5992-7283-4171-bd4d-48081329d4c8">

Reply:
Good catch! Fixed this issue in 7f58298c63c33d2a6a2a38aa74607b2f5364ef04
It now takes into account the incoming skipped items 👍